cartography/intel/gcp/util/common.py

```diff
@@ -19,3 +20,57 @@ def parse_and_validate_gcp_requested_syncs(gcp_requested_syncs: str) -> List[str
                 f'Our full list of valid values is: {valid_syncs}.',
             )
     return validated_resources
+
+def transform_iam_bindings(bindings: Dict, project_id: str) -> tuple:
```

Comment:
There are 2 use cases missing in this code.
1. code in iam load_bindings has changed drastically. Can you please compare, if we need to make any changes here?
2. deleted users, serviceAccounts, groups, etc. is handled in iam, we should ignore those entities

Reply:
Added in common.py file to ingnore deleted entities